docs/data-prep.md

```diff
@@ -103,16 +103,15 @@ Making inferences about a sample's origin is strongly dependent on the makeup of
 
 To address this, we strongly recommend adding contextual background sequences to your dataset. To make this easier, we provide a continually-updated dataset, pre-formatted for Nextstrain, through [GISAID](https://gisaid.org). To download this dataset:
 
-1. Log into GISAID's EpiCoV site
-2. Click "Downloads" to bring up a modal window
-3. In this window click on "nextmeta" to download the file `nextstrain_metadata.tsv.bz2`.
-This should be decompressed and saved as `data/global_metadata.tsv`.
-4. Then, in this window click on "nextfasta" to download the file `nextstrain_sequences.fasta.bz2`.
-This should be decompressed and saved as `data/global_sequences.fasta`.
+1. Register for a GISAID account if you don't have one already and then log into GISAID's EpiCoV database.
+2. Click "Downloads" to bring up a modal window.
+3. If you scroll down to the bottom of this modal window you should see a heading of "Genomic epidemiology" that includes entries "FASTA" and "metadata".
+4. Click on "metadata" to download a compressed file of the form `metadata_2021-04-08_08-30.tsv.gz`. Uncompress this file and save as `data/gisaid_metadata.tsv`.
```

Comment:
Just a note for future docs updates that Augur supports compressed metadata files, so at some point we could tell users to keep these files compressed.